data/json/items/tool/lighting.json

```diff
@@ -232,6 +232,178 @@
     },
     "flags": [ "LIGHT_300", "CHARGEDIM", "TRADER_AVOID", "BELT_CLIP", "WATER_BREAK", "HELMET_HEAD_ATTACHMENT", "HEAD_STRAP_MOUNT" ]
   },
+  {
+    "id": "compact_tac_flashlight",
+    "type": "TOOL",
+    "//": "Based on the TLR-1, this can go on a pistol or a long-arm",
+    "looks_like": "flashlight",
+    "name": { "str": "compact tactical flashlight (off)", "str_pl": "compact tactical flashlights (off)" },
+    "description": "A flashlight designed to endure rough conditions.  You could attach it to a firearm with a mount.  This one is shorter than a regular flashlight.",
+    "material": [ "plastic", "aluminum" ],
+    "symbol": ";",
+    "color": "black",
+    "weight": "119 g",
+    "volume": "117 ml",
+    "longest_side": "86 mm",
+    "price": 500,
+    "price_postapoc": 100,
+    "charges_per_use": 1,
+    "ammo": [ "battery" ],
+    "flags": [ "BELT_CLIP", "WATER_BREAK_ACTIVE", "HELMET_HEAD_ATTACHMENT", "HEAD_STRAP_MOUNT", "GUNLIGHT" ],
```

Comment:
>Flashlight designed to endure rough conditions
>WATER_BREAK_ACTIVE

🤔

Reply:
IPX7 is only waterproof to one meter, does the WATER_BREAK_ACTIVE flag mean it will break in the rain?  
I had it there under the assumption the flag only applies if the player was thrown into a river or other large body of water.

Reply:
If it doesn't break as is, that's fine I suppose. We don't have a good way of differentiating between a quick jaunt in a brook(which I think this IPX rating would survive) and doing over the beach silliness.